google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -30,6 +30,7 @@
    * set a lower priority for a specific RPC invocation.
    */
   public enum RpcPriority {
+    UNSPECIFIED(Priority.PRIORITY_UNSPECIFIED),
```

Comment:
This worries me a little. Adding an `enum` constant __at the beginning__ of an existing enum, means that the existing enum constants will get new ordinal values (that is; LOW was 0 before this change and will be 1 after this change), which could break existing client application code. Although I don't think it is very probable that anyone is using/checking the ordinal values of these enum constants, it is the kind of change that we should not do unless there is a very good reason to do so.

Also; do we really need this value? It seems that it is only used in the `RpcPriorityConverter`. I think that it might be better to let the `RpcPriorityConverter` return a `com.google.spanner.v1.Priority` instance (the proto enum) instead of `RpcPriority`, and then translate from `com.google.spanner.v1.Priority` to `RpcPriority` in the `ConnectionStatementExecutorImpl#statementSetRPCPriority` method (instead of converting `RpcPriority.UNSPECIFIED` to `null`).

Another reason not to add a value to this enum is that it is also used by the normal client library, so this would mean that it will be possible for a user of the normal client library to also pass in `RpcPriority.UNSPECIFIED` as the priority of a call, and I'm not sure the client library will handle that correctly.